### DISCUSSION TITLE
Assign multiple photos to a person

### DISCUSSION THREAD

**Discussion by Bishaady** [matiasdelellis/facerecognition#734] [Ideas]
Hello, 

On the app Recognize it's possible to go into a cluster and to select multiple photo and assign them to a specific person. 

![image](https://github.com/matiasdelellis/facerecognition/assets/47696152/a9dc61f3-c38a-4f94-93dc-2976704fbd9d)

Would be very nice to have this feature on Face Recognition. 
Thank you.

**Comment by matiasdelellis** [comment]
Hi,
I have it pending, the API necessary to use it already exists, but honestly the memories application has grown so much that it is difficult for me to add it. 

We need someone with good knowledge of vue and typescript.. 😞

**Comment by WhiteSockedDancer** [comment]
+1 Would love that feature.

**Comment by ftoledo** [comment]
I second this!
